modules/local/general_mergefile.nf

```diff
@@ -0,0 +1,36 @@
+process GENERAL_MERGEFILE{
+    tag "$meta.id"
+    label 'process_medium'
+    conda "bioconda::pyopenms=2.9.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/pyopenms:2.9.1--py311h9b8898c_3' :
+        'quay.io/biocontainers/pyopenms:2.9.1--py311h9b8898c_3' }"
```

Comment:
no need for `quay.io/` as it's the registry specified in the `nextflow.config` already